src/ASM/LR/ASMu2Dnurbs.h

```diff
@@ -67,6 +67,9 @@ class ASMu2Dnurbs : public ASMu2D
   //! \brief Evaluate basis functions and two derivatives in a point.
   virtual void computeBasis(double u, double v,
                             Go::BasisDerivsSf3& bas, int iel) const;
+
+private:
+  bool isNurbs;
```

Comment:
I think it would make more sense with a variable `noNurbs` which default is false, since that is the exception.
And the doxy is missing.